guides/source/api_documentation_guidelines.md

```diff
@@ -16,7 +16,8 @@ RDoc
 ----
 
 The [Rails API documentation](http://api.rubyonrails.org) is generated with
-[RDoc](http://docs.seattlerb.org/rdoc/).
+[RDoc](http://docs.seattlerb.org/rdoc/). To generate it, make sure you are
+in the `rails` root directory, run `bundle install` and execute:
```

Comment:
yea but `guides/` is an actual dir, while `rails` just refers to the folder you cloned rails into. We can't really know what it's called.